### DISCUSSION TITLE
Components not loading after cloning repo

### DISCUSSION THREAD

**Discussion by hawwokitty** [React95/React95#468] [Q&A]
Hi! I want to have the localhost of the storybook open when im working on editing the components, however when I try to open it I get a few errors. I followed the guide in the [CONTRIBUTING](https://github.com/React95/React95/blob/master/CONTRIBUTING.md) file, but it seems like its struggling with getting things from the "vite-app" file? I'm not quite sure tbh.
The first error looks like this:
`Failed to resolve import "D:React95packagescore.storybooksrcclippy-addon/preview" from "\virtual:\@storybook\builder-vite\vite-app.js". Does the file exist?`
Kind of strange that there are no backslashes in the file name? 

I thought at first maybe it just took a long time to load like someone suggested, however I don't think that's the case cuz I do get these errors... But the localhost _does_ open, the storybook is there, it's just that the components themselves are stuck in loading mode.

Could I get some guidance on exactly how I should clone the repo right for this to not happen? Or of there's a fix for the error itself? I've tried to do it both through the console with git and also with github desktop, maybe the problem happens when i install the dependencies, idk!

**Comment by ggdaltoso** [comment]
Hey, @hawwokitty 

I can't reproduce it on my machine and CI either.
I tried to fix it in #469. It would be nice if you could check it for me ❤️

**Reply by hawwokitty** [reply]
ah you're right that it seems its because of the add-ons... I tried to comment out this line:
`getAbsolutePath(__dirname, './src/clippy-addon'),`
in the main.js file of the storybook and now suddenly everything works lol! But what's really strange is that even though I commented that line out, clippy still appears 🤔

**Reply by ggdaltoso** [reply]
Alright. I just published v9.1.1 with the fix I mentioned.

Could you, please, do the check?

**Reply by hawwokitty** [reply]
that did not fix my issue unfortunately :/ i really don't know what it could be, especially since you cant replicate the issue 🤔 but for the time being i can just comment out that one line and it all seems to work fine.
